### PR TITLE
fix(*): fix rx value before sketch-to-svg export

### LINKED ISSUE #83: fix for overrounded buttons
@awinogradov  if Radius of imported element greater than smallest dimension of this element you should set it to half of this dimension

Example:

Source
image

Wrong — Current import result
image

Right
image

### PR DESCRIPTION
**Pull request by voischev**
close #83

### CODE REVIEW

File: cast/project/__screen/project__screen.js [hunk 1]
```diff
@@ -54,9 +54,18 @@ module.exports = class ProjectScreen {
         });
 
         svgo.optimize(this._svg, (result) => {
-            this._bemjson = posthtml([]).process(
-                result.data, { sync: true }
-            ).html;
+            this._bemjson = posthtml([
+                    function fixSketchRX(tree) {
+                        tree.match({ attrs: { rx: true, height: true } }, node => {
+                            let rx = node.attrs.rx;
```
Comment (Yeti-or) [inline cast/project/__screen/project__screen.js]: Could it have `ry` or it's always same?

Comment (awinogradov) [inline cast/project/__screen/project__screen.js]: Hasn't  `ry`

**Comment by awinogradov** [comment]
🔥

File: cast/project/__screen/project__screen.js (hunk 1 above)
Comment (voischev) [inline cast/project/__screen/project__screen.js]: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/rx

**Pull request merged**